practice-one/src/assets/styles/sections/footer.css

```diff
@@ -0,0 +1,71 @@
+.footer-background {
+  background-color: var(--black-color);
+}
+
+.footer-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 105px;
+  padding-bottom: 91px;
+}
+
+.footer-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  color: var(--white-color);
+}
+
+.footer-list .description {
+  opacity: 0.5;
+  color: inherit;
+}
+
+.footer-list .menu-item {
+  color: inherit;
+}
+
+.footer-item {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.item-title {
+  font-family: var(--primary-font);
+  font-weight: var(--font-weight-regular);
+  font-size: var(--font-size-lg);
+  letter-spacing: 1px;
+  color: inherit;
+  text-transform: uppercase;
+}
+
+.address {
+  display: flex;
+  flex-direction: row;
+  gap: 54px;
+}
+
+.address .description {
+  max-width: 133px;
+  max-height: 77px;
```

Comment:
Why do we need set max-height, max-width for this class?

Reply:
Because initially i also set a class for max-width and max-height but then i thought then class is not clean so i decided to include max-width and max-height in class description

Reply:
I fixed it, I create a class limit-text for set max-witdh, max-height

![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/2b064fac-8393-488d-89a6-3ef13676d723)